CONTRIBUTING.adoc

```diff
@@ -75,6 +75,12 @@ Everything related to branches, commits, and more is described #fixing-bugs-deve
 
 Where to put types and how to name them.
 
+==== Naming Convention
+
+If not explicit mentioned differently, we follow the https://en.wikipedia.org/wiki/Camel_case[`camelCase`] convention for naming classes and methods.
+This means that everything is written without spaces, but every word starts with an upper letter, e.g. `DisableIfTestFailsExtension` (class name) or `handleTestExecutionException` (method name).
```

Comment:
This is not really the case. We follow the commonly accepted Java naming conventions which include SCREAMING_SNAKE_CASE (lol) for constants.
Maybe link to this instead: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html?

Reply:
Well :p. You are arguing with the naming of constants, while the text clearly is written about 

> convention for naming classes and methods.

And the  Java naming conventions also say camelCase :D

> Class names should be nouns, in mixed case with the first letter of each internal word capitalized

But yeah you are right, the Java code conventions would be the correct source. Will update it later.